bundles/net.tourbook.web/src/net/tourbook/web/WebContentServer.java

```diff
@@ -407,7 +407,7 @@ private static void handle_404(final HttpExchange httpExchange, final File file,
          os = httpExchange.getResponseBody();
          os.write(response.getBytes());
 
-         StatusUtil.log(response + " " + file.getPath());//$NON-NLS-1$
+         StatusUtil.log(response + UI.SPACE1 + file.getPath());
```

Comment:
The `net.tourbook.web` plugin should currently not be touched

Reply:
Ok, fixed